core/src/androidTest/java/org/hisp/dhis/android/core/common/CategoryOptionMockFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.hisp.dhis.android.core.common;
+
+
+import org.hisp.dhis.android.core.category.CategoryOption;
+
+import java.util.Date;
+
+public final class CategoryOptionMockFactory {
```

Comment:
this class is not a factory, it's an [object mother](https://martinfowler.com/bliki/ObjectMother.html), can you refactor it?

Reply:
Sure!